Subscriptions (#91)
AntennaPod/antennapod.github.io: _documentation/subscriptions/subscription-groups.md

---

> title: Subscription groups / tags / categories

Reviewer: This appears also in the breadcrumb, and it's a bit confusing with the slashes.
```suggestion
title: Subscription tags (aka groups or categories)
```
Ideally I would leave out 'groups' and 'categories' altogether, but I realise it'll help people find this article.

layout: doc
level: "2"
group: "subscriptions"
icon: "fas fa-tags"
---

# Assigning groups / tags / categories to subscription

> To manage subscriptions it is possible to assign groups / tags / categories to a subscription.

Reviewer: ```suggestion
To manage subscriptions it is possible to assign tags (sometimes called groups or categories) to a subscription.
```

> Please note that for the moment it is not possible to create queue(s) based on it. To do so it would first be needed to support multiple queues. (See related [issue 2648](https://github.com/AntennaPod/AntennaPod/issues/2648))

Reviewer: I would put this in a separate paragraph (by introducing an extra white line. And some other changes:
```suggestion
Please note that for the moment it is not possible to create queues based on tags. This would require support for multiple queues first. (See also the [feature request](https://github.com/AntennaPod/AntennaPod/issues/2648) on our development platform.)
```

> To assign groups / tags / categoriers to a podcasdt you have to :

Reviewer 2: I would try to avoid the word "podcast". While it usually refers to the subscription, more and more users use the same word to refer to episodes. When only using subscription/episode, it is more clear what elements can be tagged.

```suggestion
To assign groups / tags / categories to a subscription you have to :
```

Reviewer: I disagree. A podcast is a podcast, an episode an episode. This is notably different from the situation with 'blog' and 'blog (post)'. If institutions aren't using the right terminology they will only help the wider public (who, let's face it, are only starting to get introduced to the concept) to not get familiar with the proper terms.
And the confusion is minor, without the risk of causing severe consequences (e.g. data loss).

> To assign groups / tags / categories to a subscription you have to :

Reviewer: ```suggestion
To assign tags to a podcast you have to :
```

> - go to podcast / subscription screen

Reviewer: ```suggestion
- open a specific podcast
```

(main point here was to remove 'subscription' for consistency)

> - tap on gear icon to podcast access specific settings

Reviewer: ```suggestion
- tap on the `gear` icon to access its settings
```

> - tap on group entry

Reviewer: ```suggestion
- tap on the `Tags` setting
```
